Lib/test/test_datetime.py

```diff
@@ -20,7 +20,7 @@
 # XXX(gb) First run all the _Pure tests, then all the _Fast tests.  You might
 # not believe this, but in spite of all the sys.modules trickery running a _Pure
 # test last will leave a mix of pure and native datetime stuff lying around.
-test_classes = []
+all_test_classes = []
```

Comment:
Good point. I'll check which versions are affected (i.e. the branches on the `cpython` repository, right?) and open another bug. What should I do with this fix, though? 

If I let it remain here, it may potentially lead to a merge conflict?